stable/sematext-agent/README.md

```diff
@@ -0,0 +1,76 @@
+
+|           Parameter           |             Description              |                  Default                  |
+|-------------------------------|--------------------------------------|-------------------------------------------|
+| `containerToken`              | Sematext Container token             | `Nil` Provide your Container token        |
+| `logsToken`                   | Sematext Logs token                  | `Nil` Provide your Logs token             |
+| `region`                      | Sematext region                      | `US` Sematext US or EU region             |
+| `agent.image.repository`      | The image repository                 | `sematext/agent`                          |
+| `agent.image.tag`             | The image tag                        | `latest`                                  |
+| `agent.image.pullPolicy`      | Image pull policy                    | `Always`                                  |
+| `agent.service.port`          | Service port                         | `80`                                      |
+| `agent.service.type`          | Service type                         | `ClusterIP`                               |
+| `agent.resources`             | Agent resources                      | `{}`                                      |
+| `logagent.image.repository`   | The image repository                 | `sematext/logagent`                       |
+| `logagent.image.tag`          | The image tag                        | `latest`                                  |
+| `logagent.image.pullPolicy`   | Image pull policy                    | `Always`                                  |
+| `logagent.resources`          | Logagent resources                   | `{}`                                      |
+| `customUrl.metricsServer`     | Custom endpoint for Metrics receiver | `Nil`                                     |
+| `customUrl.logsServer`        | Custom endpoint for Logs receiver    | `Nil`                                     |
+| `customUrl.eventServer`       | Custom endpoint for Event receiver   | `Nil`                                     |
+| `serviceAccount.create`       | Create a service account             | `true`                                    |
+| `serviceAccount.name`         | Service account name                 | `Nil` Defaults to chart name              |
+| `rbac.create`                 | RBAC enabled                         | `true`                                    |
+| `tolerations`                 | Tolerations                          | `[]`                                      |
+| `nodeSelector`                | Node selector                        | `{}`                                      |
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example:
+
+```bash
+$ helm install --name release_name \
+    --set containerToken=YOUR_CONTAINER_TOKEN,region=EU \
```

Comment:
I would not add the region key in this example given that you call it out to be added above. It just seems inconsistent with the previous installation details above

```suggestion
    --set containerToken=YOUR_CONTAINER_TOKEN \
```

Reply:
I agree it is not consistent. Fixed.